src/Features/VisualBasic/GenerateType/VisualBasicGenerateTypeService.vb

```diff
@@ -90,10 +90,6 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.GenerateType
                 document As SemanticDocument, simpleName As SimpleNameSyntax, cancellationToken As CancellationToken, ByRef generateTypeServiceStateOptions As GenerateTypeServiceStateOptions) As Boolean
             generateTypeServiceStateOptions = New GenerateTypeServiceStateOptions()
 
-            If simpleName.GetAncestorOrThis(Of ImportsStatementSyntax)() IsNot Nothing Then
```

Comment:
I think it's low-pri and it raises the question of whether to generate a class or a module and then whether Generate member suggestions for simple names in the file from then on will have as an option to generate those members inside of the imported class/module. The big use case for GFU is consumption first programming. But if all I can do is generate a container type but nothing that I'd actually consume from the container then it's a bit of a dead-end feature.